error.go

```diff
@@ -45,3 +46,22 @@ func NewError(expectedRespCode []int, resp *http.Response) Error {
 		response:             resp,
 	}
 }
+
+// SError ...
+type SError struct {
+	T   ScreenType
+	Err Error
+}
+
+// ScreeningError ...
+type ScreeningError struct {
```

Comment:
Let's change this to `type ScreeningErrors []map[ScreeningType]error` or even just `type ScreeningErrors []error`. That way, when `GetScreenings` is called, we can type case the `error` to `ScreeningErrors` and perform a loop and then type cast the error type to possibly a `ResponseError`.